index.yaml

```diff
@@ -409,15 +464,112 @@ entries:
       catalog.cattle.io/release-name: fleet-crd
     apiVersion: v2
     appVersion: 0.3.0
-    created: "2021-10-18T13:41:39.927045-07:00"
+    created: "2021-09-30T14:07:17.907549299-04:00"
     description: Fleet Manager CustomResourceDefinitions
     digest: 08e3af78da30602b47b60ebfb8e509703dbedc5b312f6aa3a9e9b0275adca75a
     icon: https://charts.rancher.io/assets/logos/fleet.svg
     name: fleet-crd
     urls:
     - assets/fleet-crd/fleet-crd-0.3.000.tgz
     version: 0.3.000
+  harvester-cloud-provider:
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: Harvester Cloud Provider
+      catalog.cattle.io/kube-version: '>= 1.18'
+      catalog.cattle.io/namespace: kube-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/rancher-version: '>= 2.6.1-0 <= 2.6.99-0'
+      catalog.cattle.io/release-name: harvester-cloud-provider
+      catalog.cattle.io/ui-component: harvester-cloud-provider
+      catalog.cattle.io/upstream-version: 0.1.8
+    apiVersion: v2
+    appVersion: v0.1.1
+    created: "2021-12-14T23:35:42.25204+08:00"
+    description: A Helm chart for Harvester Cloud Provider
+    digest: 0724179c7afb355fa66a9ae09ea067c82c0d23be82f13c1ab7af4d53867a9757
+    keywords:
+    - infrastructure
+    - harvester
+    maintainers:
+    - name: harvester
+    name: harvester-cloud-provider
+    type: application
+    urls:
+    - assets/harvester-cloud-provider/harvester-cloud-provider-100.0.0+up0.1.8.tgz
+    version: 100.0.0+up0.1.8
+  harvester-csi-driver:
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: Harvester CSI Driver
+      catalog.cattle.io/kube-version: '>= 1.18'
+      catalog.cattle.io/namespace: kube-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/rancher-version: '>= 2.6.1-0 <= 2.6.99-0'
+      catalog.cattle.io/release-name: harvester-csi-driver
+      catalog.cattle.io/ui-component: harvester-csi-driver
+      catalog.cattle.io/upstream-version: 0.1.9
+    apiVersion: v2
+    appVersion: v0.1.1
+    created: "2021-12-14T23:36:22.530546+08:00"
+    description: A Helm chart for Harvester CSI driver
+    digest: 2173919998b23049fcb928f358c6e933818c9c3210b7332dfd0270bbe8c8aaa8
+    keywords:
+    - infrastructure
+    - harvester
+    maintainers:
+    - name: harvester
+    name: harvester-csi-driver
+    type: application
+    urls:
+    - assets/harvester-csi-driver/harvester-csi-driver-100.0.0+up0.1.9.tgz
+    version: 100.0.0+up0.1.9
```

Comment:
we are releasing two charts with the same version: `harvester-csi-driver-100.0.0+up0.1.8` and `harvester-csi-driver-100.0.0+up0.1.9`
I guess the 0.1.8 one should be removed?  need to confirm with the harvester team.